sites/cheerpx/src/pages/index.astro

```diff
@@ -1,60 +1,183 @@
+					CheerpX is a JavaScript library that enables secure <b
+						>client-side execution of x86 binaries</b
+					> on any browser. Currently available only on its beta version and developed
+					by Leaning Technologies.
+				</p>
+				<div class="flex gap-4 justify-left mt-12">
+					<LinkButton
+						type="primary-cheerpx"
+						href="https://webvm.io/"
+						label="Try the demo"
+					/>
+					<LinkButton
+						type="secondary-cheerpx"
+						href="/docs/getting-started"
+						iconRight="mi:arrow-right"
+						label="Get started"
+					/>
+				</div>
+			</div>
+			<div class="hidden md:block flex justify-center 2xl:pl-56">
+				<Image src={HeroAsset} alt="Cheerp Logo in a cube" class="" />
+			</div>
+		</div>
+	</section>
+	<section
+		class="justify-center py-20 bg-black px-2 sm:px-2 md:pt-16 md:px-8 lg:px-4 lg:pt-32 xl:pt-42 xl:px-56 2xl:px-60 2xl:pt-60"
+	>
+		<div class="flex justify-center items-center">
+			<Image src={CalendarAsset} alt="Sourcerer" class="w-48" />
+		</div>
+		<h1
+			class="text-white text-4xl md:text-5xl lg:text-6xl xl:text-7xl font-semibold text-center text-wrap"
+		>
+			<span
+				class="bg-gradient-to-r from-[#446887] to-[#909090] via-[#6386a5] text-transparent bg-clip-text"
+				>Coming soon</span
 			>
-				<b>Virtual machines</b> in your web app.
-			</h1>
-			<p class="text-lg text-balance">
-				CheerpX is a JavaScript library that enables secure client-side
-				execution of x86 binaries on any browser.
-			</p>
-			<p class="text-lg text-balance text-stone-400">Coming soon.</p>
+		</h1>
+		<p class="text-center text-xl font-light text-stone-300 py-4 px-30">
+			CheerpX official release is expected for late 2024. For now, enjoy the
+			beta trial and our popular WebVM demo.
+		</p>
+		<div
+			class="grid grid-cols-3 py-8 text-stone-300 px-2 sm:px-8 md:px-16 lg:px-32 xl:px-28 2xl:px-72"
+		>
+			<div class="text-center hover:text-yellow-600">
+				<a href="https://github.com/leaningtech/webvm/forks">
+					<p class="text-2xl font-bold">900+</p>
+					<p>Forks on our WebVM repository</p>
+				</a>
+			</div>
+			<div class="text-center hover:text-yellow-600">
+				<a href="https://github.com/leaningtech/webvm/stargazers">
+					<p class="text-2xl font-bold">4.5k+</p>
+					<p>Stars on our demo GitHub</p>
+				</a>
+			</div>
+			<div class="text-center hover:text-yellow-600">
+				<a href={DISCORD_URL}>
+					<p class="text-2xl font-bold">1k+</p>
+					<p>Discord members</p>
+				</a>
+			</div>
 		</div>
-		<div class="flex items-center gap-4 justify-center mt-12">
-			<LinkButton
-				type="discord"
-				href="https://discord.leaningtech.com"
-				iconLeft="fa-brands:discord"
-				label="Subscribe for updates"
-			/>
-			<LinkButton
-				type="secondary"
-				href="https://webvm.io"
-				iconRight="mi:arrow-right"
-				label="Try the demo"
-			/>
+		<div class="flex justify-center items-center">
+			<LinkButton type="primary-cheerpx" href="#youtube" label="Learn more" />
 		</div>
```

Comment:
Below are just some advice:

Because the text above says ‘For now, enjoy the beta trial and our popular WebVM demo.’ I think it would be good to add another button like ‘try our WebVM demo’ here next to the learn more button to attract people to click on it.

Also, we can edit the text to simply describe the Games Runner Extension and add another ‘Try our Games Runner Extension’ button here as well.

(Or, If there's enough bandwidth, we can also definitely list WebVM and Games Runner Extension in a separate section)